internal/bundlereader/resources.go

```diff
@@ -167,7 +160,7 @@ func addRemoteCharts(directories []directory, base string, charts []*fleet.HelmO
 				auth = Auth{}
 			}
 
-			chartURL, err := chartURL(chart, auth)
+			chartURL, err := ChartURL(*chart, auth)
```

Comment:
It's unclear to me why we need to export this function, which is not called outside this package.